modules/contentbox/modules/contentbox-admin/layouts/inc/HTMLHead.cfm

```diff
@@ -36,14 +36,15 @@
     <!--- ********************************************************************* --->
     <!---                           CSS                                           --->
     <!--- ********************************************************************* --->
-	<link rel="stylesheet" href="#prc.cbroot#/includes/css/contentbox.min.css">
+	<link rel="stylesheet" href="#cbadminElixirPath( fileName='css/contentbox.css' )#">
+	<link rel="stylesheet" href="#cbadminElixirPath( fileName='css/theme.css' )#">
```

Comment:
Any reason why the split?

Reply:
The old Gruntfile was compiling them as split and then merging them.  I can consolidate them back again, though.  That's easy enough.

Reply:
Done